test/resources/SimpleAudioPlayer.js

```diff
@@ -44,10 +44,10 @@ SimplePlayer.prototype.handle = function () {
 
         if (intent.name === "Play") {
             this.play(podcastFeed[podcastIndex], 0, "REPLACE_ALL", podcastIndex);
-
-        } if (intent.name === "Ignore") {
```

Comment:
this line used to be a new if statement, was that intended? I changed it and my new intent to `else if`